libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway.hs

```diff
@@ -489,3 +490,21 @@ instance
     first (\case {})
       . computationResultToEither
       $ Agda.epochStep env st sig
+
+instance
+  IsConwayUniv fn =>
+  ExecSpecRule fn "NEWEPOCH" Conway
+  where
+  type ExecContext fn "NEWEPOCH" Conway = [GovActionState Conway]
+  type ExecEnvironment fn "NEWEPOCH" Conway = EpochExecEnv Conway
+
+  environmentSpec _ = epochEnvSpec
+
+  stateSpec _ _ = newEpochStateSpec
+
+  signalSpec _ _ _ = epochSignalSpec
```

Comment:
I used the `EPOCH` specs of the environment and the signal for now since the types are equivalent.